web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -16,7 +17,7 @@
     z-index: 1000;
     position: fixed;
     box-shadow: 0 0 3px rgba(0,0,0,.3);
-    background: rgba(255, 0, 0, .6);;
+    background: rgba(255,0,0,.6);
```

Comment:
TBH if we were going to alter the diff to do anything with the commas, it would be better to add commas to the line above and leave the ones that were here. Is there a specific reason for removing them?